kubernetes/spec/config/incluster_config_spec.rb

```diff
@@ -50,6 +54,18 @@
 
       incluster_config.configure(actual)
       expect(actual).to be_same_configuration_as(expected)
+
+      old_expired_at = incluster_config.token_expires_at
+      incluster_config.instance_variable_set(
+        :@token_file,
+        Kubernetes::Testing.file_fixture('tokens/token2').to_s
+      )
+      sleep(5)
```

Comment:
I'd really prefer not to have sleeps in tests. can we inject the clock instead?

Reply:
done